### DISCUSSION TITLE
Starter Template: Dart Shelf

### DISCUSSION THREAD

**Discussion by hotdang-ca** [BattlesnakeOfficial/feedback#245] [Q&A] [answered]
I'm looking to submit a new Community Starter Template for [Dart](https://dart.dev) and the [Shelf framework](https://pub.dev/packages/shelf). The contribution guidelines link to the Feedback thread (locked), and the PR process (which just goes to the NodeJS official project).

How can I proceed to share this Community Starter Template for a language growing in popularity?

**Comment by bvanvugt** [comment, marked as answer]
Cool! I'll see if I can update those links - thanks for letting me know.

Process is pretty simple - just submit a PR to add your projects to this file:
https://github.com/BattlesnakeOfficial/docs/blob/main/docs/starter-projects.md?plain=1

I can review it this week and get it deployed pretty quick :+1:

**Reply by hotdang-ca** [reply]
Pull request created https://github.com/BattlesnakeOfficial/docs/pull/50

Thanks!
